src/uthash.h

```diff
@@ -255,34 +255,34 @@ do {
 #define HASH_GHV_COMPUTE(hh, keyptr, keylen_in, add)                             \
   HASH_FCN((keyptr), (keylen_in), (add)->hh.hashv)
 
-#define HASH_ADD_SORTED(hh,head,keyptr,keylen_in,hashval,add,cmpfcn)             \
+#define HASH_SORTED_ADD(hh,head,keyptr,keylen_in,hashval,add,cmpfcn)             \
```

Comment:
Reopening the bikeshed as long as the API isn't nailed down yet: How would you feel about naming this family of functions `HASH_ADD_INORDER` and the previous `BY_HASH_VALUE`/`HVAL` family `HASH_ADD_BYHASHVALUE`, with combinations like `HASH_ADD_BYHASHVALUE_INORDER`?